nextcord/message.py

```diff
@@ -543,6 +545,56 @@ def flatten_handlers(cls):
     return cls
 
 
+class MessageInteraction:
+    """Represents a message's interaction data.
+
+    A message's interaction data is a property of a message when the message
+    is a response to an interaction from any bot.
+
+    .. versionadded:: 2.0
```

Comment:
I believe this will be `2.1` now? Right @ooliver1

Reply:
Correct, sorry @TrapDrap but could you edit these versionadded please

Reply:
changed